modules/apps/portal/portal-http-impl/src/test/java/com/liferay/portal/http/internal/HttpImplTest.java

```diff
@@ -63,6 +144,98 @@ public void testIsNonProxyHost() throws Exception {
 		}
 	}
 
+	private Tuple _getHttpConnectionStrategies() {
+		ClientExecChain clientExecChain = ReflectionTestUtil.getFieldValue(
+			(CloseableHttpClient)ReflectionTestUtil.invoke(
+				_httpImpl, "getCloseableHttpClient",
+				new Class<?>[] {HttpHost.class}, new Object[] {null}),
+			"execChain");
+
+		while (true) {
+			clientExecChain = ReflectionTestUtil.getFieldValue(
+				clientExecChain, "requestExecutor");
+
+			if (clientExecChain instanceof MainClientExec) {
+				return new Tuple(
+					ReflectionTestUtil.getFieldValue(
+						clientExecChain, "keepAliveStrategy"),
+					ReflectionTestUtil.getFieldValue(
+						clientExecChain, "reuseStrategy"));
+			}
+		}
+	}
+
+	private void _setHttpKeepAliveTimeout(int keepAliveTimeout) {
```

Comment:
@ericyanLr Please remove this method, just invoke the put directly in your test logic